src/Containers/CadView.jsx

```diff
@@ -301,6 +312,9 @@ export default function CadView({
 
   /** When viewer is ready, load IFC model. */
   async function onViewer() {
+    if (isOPFSAvailable === null) {
```

Comment:
assertDefined instead

Reply:
I don't think we want to assertDefined there, as onViewer can be called possibly after isOPFSAvailable resolves, in which case it will still be null.

Reply:
Ok, then return